src/main.ts

```diff
@@ -1,13 +1,19 @@
 import App from './App.svelte';
 
+// You can define default input values here. The values here will be used if no value is given. 
+const inputs = { 
+  ...{
```

Comment:
Defining an object and then instantly spreading it seems a bit dank. Couldn't we just do?

```
const props = {
  foo: 'bar',
   ...window.templateProps
}
```